pkg/command/autoscaling/update_test.go

```diff
@@ -118,7 +122,8 @@ func TestNewAsUpdateSetCommand(t *testing.T) {
 		}
 		client := k8sfake.NewSimpleClientset(cm)
 		p := pkg.AdminParams{
-			ClientSet: client,
+			ClientSet:          client,
+			InstallationMethod: pkg.InstallationMethodStandalone,
```

Comment:
@lanceliuu I think it's good to add at least one negative case here to test InstallationMethod, perhaps set it to InstallationMethodOperator mode